core/primitives-core/src/version.rs

```diff
@@ -184,6 +185,7 @@ impl ProtocolFeature {
             ProtocolFeature::RestrictTla
             | ProtocolFeature::TestnetFewerBlockProducers
             | ProtocolFeature::SimpleNightshadeV2 => 64,
+            ProtocolFeature::SimpleNightshadeV3 => 65,
```

Comment:
To be noted, STABLE_PROTOCOL_VERSION is already set as 65 on master. We may need to change this while cherry picking?